### PR TITLE
Harvest of Features from ISS Clinic Needs + Miscellaneous Fixes

### PR DESCRIPTION
**Pull request by ssmusoke**
1. #2 - Add a concept for pill balances and update the ART encounter page with it 

2. Correct the concept for the coreapps sticky note feature from Ref App 2.7 to CIEL:162169

3. Dashboard widget additions:
- ART Status: Baseline WHO HIV CLINICAL STAGING
- Last ART Status: WHO HIV CLINICAL STAGE

4. Update of Reporting Compatibility module to fix issue with deleting data exports and saved searches https://trello.com/c/93fLIp5J

5. Add Flag to show patients who have a current un-supressed viral load

6. Clean up the alignment for OIs, Potential Side effects and other medication provided https://trello.com/c/7N6KO7oP

7. Other Gender and Other Occupation concepts as needed by comment #1 on https://github.com/METS-Programme/openmrs-module-ugandaemr-iss/pull/2

8. Updates to module versions to those in Reference Application 2.8 most significantly HTML Form Entry which has the functionality to block multiple encounters of the same type being entered on the same day

9. Add a fix for numeric data saved for ART Care grouping concept 99162 on the ART summary page - see Trello card https://trello.com/c/lJkfbxqG - this causes an error when saving the ART summary page

### CODE REVIEW

**Comment by ssmusoke** [comment]
@slubwama for your review

**Comment by ssmusoke** [comment]
@slubwama bringing this up to the top of your mailbox - I would like to use this for a package for ISS - just got a call

**Comment by slubwama** [comment]
Test Failing to build
https://pastebin.com/c4yMYzyY

**Comment by ssmusoke** [comment]
@slubwama Added fix - looks like I had lost it in one of the merges

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -8,6 +8,19 @@
         See http://www.liquibase.org/manual/home#available_database_refactorings
         for a list of supported elements and attributes
     -->
+	<changeSet id="ugandaemr-05092018-1043" author="ssmusoke">
+		<comment>Fix bad data for the ART care section which is causing an error when saving the summary page</comment>
+		<sql>
+			UPDATE obs SET value_numeric = NULL WHERE concept_id = 99162 AND value_numeric IS NOT NULL;
+		</sql>
+	</changeSet>
+	<changeSet id="ugandaemr-iss-19072018-1002" author="ssmusoke">
```
Comment (slubwama) [inline api/src/main/resources/liquibase.xml]: ugandaemr-iss-19072018-1002 to appropriate

**Pull request merged**